packages/eas-cli/src/commandUtils/pagination.ts

```diff
@@ -27,19 +27,27 @@ const parseFlagInputStringAsInteger = (
   return inputAsNumber;
 };
 
+export const getLimitFlagWithCustomValues = ({
+  defaultTo,
+  limit,
+}: {
+  defaultTo: number;
+  limit: number;
+}): any =>
+  Flags.integer({
+    description: `The number of items to fetch each query. Defaults to ${defaultTo} and is capped at ${limit}. Using a lower value may help increase command speed.`,
+    // eslint-disable-next-line async-protect/async-suffix
+    parse: async input => parseFlagInputStringAsInteger(input, 'limit', 1, limit),
+  });
+
```

Comment:
turns out with the introduction of update groups we now have different limits per query object. branches/updates are capped at 100, update groups can be 2x the size, so they're capped at 50. 

I added a configurable limit flag here to match that. Alternatively we could:
- raise/lower the entity limit on www for all types
- match all limits here on CLI to that